tsec-libsodium/src/main/scala/tsec/libsodium/kx/KeyExchange.scala

```diff
@@ -0,0 +1,57 @@
+package tsec.libsodium.kx
+
+import cats.effect.Sync
+import tsec.libsodium.ScalaSodium
+import tsec.libsodium.cipher.SodiumKey
+import tsec.libsodium.pk.{PrivateKey, PublicKey, SodiumKeyPair}
+
+object KeyExchange {
+  sealed trait KeyExchange
```

Comment:
this should be _outside_ the package object.

Reply:
is there a benefit of doing this or just convention ?

Reply:
Convention